gimme_aws_creds/config.py

```diff
@@ -145,6 +147,8 @@ def get_args(self):
             '--action-setup-fido-authenticator', action='store_true',
             help='Sets up a new FIDO WebAuthn authenticator in Okta'
         )
+        parser.add_argument('--okta-app', type=int)
```

Comment:
I added two of new arguments; `--okta-app`, `--okta-role` are automatically select the specific app and role.

Reply:
the inputs should be the app and role names, not their position in the list from Okta.  Those positions will change as accounts/roles are added and removed.

Reply:
also, there are existing config options for these settings (`aws_appname` and `aws_rolename`) the parameters should match those names